### PR TITLE
Explore logs styling

### PR DESCRIPTION
**Pull request by torkelo**
fixes the spacing between levels introduced in #14352 

and adds a subtle hover effect. Would like to try different hover effects / backgrounds / borders, but not sure I have time to spend on this. Feel the look of the logs rows is still ugly (especially labels), but making minor progress here. 

I did experiment with adding a transform: scale(1.005) to the over effect, it has a really good effect of making the row/time/level standout but make the hover effect when you move the mouse around more distracting, so did not include that. 

image

### CODE REVIEW

File: public/sass/components/_panel_logs.scss [hunk 1]
```diff
@@ -93,36 +110,56 @@
...
+  // width: 12px;
+
+  &::after {
+    content: '';
+    display: block;
+    position: absolute;
+    top: 1px;
+    bottom: 1px;
+    width: 3px;
+    background-color: $gray-2;
```
Comment (davkal) [inline public/sass/components/_panel_logs.scss]: This no longer matches the log graph label color for `Unknown` log levels.

**Comment by davkal** [comment]
Small nit, rest LGTM

File: public/sass/components/_panel_logs.scss (hunk 1 above)
Comment (torkelo) [reply public/sass/components/_panel_logs.scss]: ah, yes. where is that set? needs to be theme aware probably

**Comment by davkal** [comment]
Unknown label color still does not match in light theme:
screenshot 2018-12-06 at 21 33 30

I'm running light theme as `env GRAFANA_THEME=light yarn start`.

**Comment by torkelo** [comment]
Try setting light theme in user preferences, your hot reload theme is overriding what theme grafana thinks it’s showing

**Pull request merged**